src/core/bpf-firewall.c

```diff
@@ -795,6 +795,12 @@ int bpf_firewall_reset_accounting(int map_fd) {
 
 static int bpf_firewall_unsupported_reason = 0;
 
+#define log_unsupported(err, msg)                                       \
+        ({                                                              \
+                bpf_firewall_unsupported_reason = log_debug_errno(err, msg); \
+                supported = BPF_FIREWALL_UNSUPPORTED;                   \
+        })
+
```

Comment:
given this is kinda a "local" macro, maybe actually define it within the boundaries of the bpf_firewall_supported() declaration